websites/D/discord.style/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "bris",
+    "id": "656165298818514946"
+  },
+  "service": "discord.style",
+  "description": {
+    "en": "Online library for finding and sharing free Discord Templates. Upload your own Discord Template and collect votes!"
+  },
+  "url": "www.discord.style",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/Fh7T0b6.png",
```

Comment:
isn't there better quality logo?

Reply:
bumping.

Reply:
I guess not, I couldn't find a better logo.

Reply:
![logo](https://i.imgur.com/YHxQl2n.png)